internal/server/data/data.go

```diff
@@ -179,16 +179,16 @@ func deleteAll[T models.Modelable](db *gorm.DB, selectors ...SelectorFunc) error
 	return db2.Delete(new(T)).Error
 }
 
-func Count[T models.Modelable](db *gorm.DB, selectors ...SelectorFunc) (*int64, error) {
+func Count[T models.Modelable](db *gorm.DB, selectors ...SelectorFunc) (int64, error) {
 	db2 := db
 	for _, selector := range selectors {
 		db2 = selector(db2)
 	}
 
 	var count int64
-	if err := db.Model((*T)(nil)).Count(&count).Error; err != nil {
-		return nil, err
+	if err := db2.Model((*T)(nil)).Count(&count).Error; err != nil {
```

Comment:
db2 wasn't used before so selectors were not applied. Haven't run into this previously since most calls to `Count` don't use selectors

Reply:
That was in the original version of selectors but if I recall there were some weird side effects which is why we're using a temporary var for the db here